### DISCUSSION TITLE
Error executing `writeFile`

### DISCUSSION THREAD

**Discussion by ayangweb** [tauri-apps/tauri#11341] [Q&A]
The `writeFile` execution reported an error, I suspect that it is because the path is a path containing a `.` path, how can I fix this? I am using `appDataDir` path!

<img width="733" alt="image" src="https://github.com/user-attachments/assets/9893fd77-fa27-42b2-b49d-0270c1b001b5">

**Comment by FabianLars** [comment]
Moved to https://github.com/tauri-apps/plugins-workspace/issues/1927
